packages/llama-index-auth0-ai/README.md

```diff
@@ -1,6 +1,6 @@
 # Auth0 AI for LlamaIndex
 
-This package integrates [LlamaIndex](https://docs.llamaindex.ai/en/stable/) with [Auth0 AI](https://www.auth0.ai/) for enhanced document retrieval capabilities.
+`llama-index-auth0-ai` is an SDK for building secure AI-powered applications using [Auth0](https://www.auth0.ai/), [Okta FGA](https://docs.fga.dev/) and [LlamaIndex](https://docs.llamaindex.ai/en/stable/).
```

Comment:
should this not be `llamaindex-auth0-ai` ?

Reply:
maybe @jcenturion can answer this (when I started working on this, these packages had already been created)

Reply:
never mind, other packages are named the same:
![image](https://github.com/user-attachments/assets/893eb926-1c96-4173-ba61-63337ebcdb5e)